### DISCUSSION TITLE
Sort locales in installer

### DISCUSSION THREAD

**Discussion by abcdenis** [dbeaver/dbeaver#14117] [General]
Hi,
there is a step on installation where one have to select locale.
Dropdown list contains unsorted locale names so it is difficult a bit to choose correct locale.
It will be great to sort it.
Thank you.
![Снимок экрана 2021-09-24 в 23 00 41](https://user-images.githubusercontent.com/3295735/134733483-17d07f23-5fd9-4973-9c06-6bbcad39e413.png)

**Comment by ShadelessFox** [comment]
Hello @abcdenis,

Are you sure your issue is related to DBeaver? We don't support that much languages. Also, I can see <kbd>VMWare</kbd> label in the corner.
